Result/AbstractResultsIterator.php

```diff
@@ -310,6 +310,24 @@ public function getDocumentScore()
     }
 
     /**
+    * Returns sort of current hit.
+    *
+    * @return array
+    */
+    public function getDocumentSort()
```

Comment:
I'm not sure the name is correct. `getSort()` as provided in issue looks much better.